packages/edge-config/src/index.ts

```diff
@@ -66,147 +64,169 @@ interface EdgeConfigClientOptions {
+          return loaders.has.load(key);
+        },
+        {
+          name: 'has',
+        },
+      ),
+      getMany: trace(
+        async function getMany<T = (EdgeConfigValue | undefined)[]>(
+          keys: string[],
+        ): Promise<T> {
+          const loaders = getLoadersInstance(
+            loaderOptions,
+            loadersInstanceCache,
+          );
 
-        assertIsKeys(keys);
-        const values = await loaders.get.loadMany(keys);
+          assertIsKeys(keys);
+          const values = await loaders.get.loadMany(keys);
 
-        // throw error in case the edge config could not be found
-        const error = values.find((v): v is Error => v instanceof Error);
-        if (error) throw error;
+          // throw error in case the edge config could not be found
+          const error = values.find((v): v is Error => v instanceof Error);
+          if (error) throw error;
 
-        // prime get() and has() calls with the result of getMany()
-        keys.forEach((key, index) => {
-          if (!key) return;
-          const value = values[index];
-          loaders.get.prime(key, value);
-          loaders.has.prime(key, value !== undefined);
-        });
-
-        return clone(values) as T;
-      },
-      {
-        name: 'getMany',
-      },
-    ),
-    getAll: trace(
-      async function getAll<T = EdgeConfigItems>(
-        keys?: (keyof T)[],
-      ): Promise<T> {
-        const loaders = getLoadersInstance(loaderOptions, loadersInstanceCache);
-        if (keys === undefined) {
-          const items = await loaders.getAll.load('#').then(clone);
-
-          // prime get() and has() calls with the result of getAll()
-          Object.entries(items).forEach(([key, value]) => {
+          // prime get() and has() calls with the result of getMany()
+          keys.forEach((key, index) => {
+            if (!key) return;
+            const value = values[index];
             loaders.get.prime(key, value);
-            loaders.has.prime(key, true);
+            loaders.has.prime(key, value !== undefined);
           });
 
-          return items as T;
-        }
+          return clone(values) as T;
+        },
+        {
+          name: 'getMany',
+        },
+      ),
+      getAll: trace(
+        async function getAll<T = EdgeConfigItems>(
+          keys?: (keyof T)[],
+        ): Promise<T> {
+          const loaders = getLoadersInstance(
+            loaderOptions,
+            loadersInstanceCache,
+          );
+          if (keys === undefined) {
+            const items = await loaders.getAll.load('#').then(clone);
 
-        assertIsKeys(keys);
-        const values = await loaders.get.loadMany(keys);
+            // prime get() and has() calls with the result of getAll()
+            Object.entries(items).forEach(([key, value]) => {
+              loaders.get.prime(key, value);
+              loaders.has.prime(key, true);
```

Comment:
In `get` we're doing `value !== undefined` so I think we should prime the same way here

```suggestion
              loaders.has.prime(key, value !== undefined);
```

Reply:
Don't think I've changed that. Maybe better to address in the `batching-and-deduping` branch directly?